test/unit/alphabet/quality/qualified_test.cpp

```diff
@@ -35,12 +35,27 @@
 #include <gtest/gtest.h>
 
 #include <seqan3/alphabet/concept.hpp>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/alphabet/quality/phred42.hpp>
+#include <seqan3/alphabet/quality/aliases.hpp>
 #include <seqan3/alphabet/quality/qualified.hpp>
 
+#include "../alphabet_test_template.hpp"
+#include "../alphabet_constexpr_test_template.hpp"
+
 using namespace seqan3;
 
+using qualified_types = ::testing::Types<qualified<dna4, phred42>,
+                                         qualified<dna4, phred63>,
+                                         qualified<aa27, phred42>,
+                                         qualified<gapped<dna4>, phred42>,
+                                         dna4q>;
+
+INSTANTIATE_TYPED_TEST_CASE_P(qualified, alphabet, qualified_types);
+INSTANTIATE_TYPED_TEST_CASE_P(qualified, alphabet_constexpr, qualified_types);
+
```

Comment:
Sry, I don't understand your question.

Reply:
You forgot the quality tests for the qualified alphabets:
```cpp
INSTANTIATE_TYPED_TEST_CASE_P(qualified, quality, quality_types);
```

Reply:
Ah Okay! Yeah, because the original `quality_test` didn't include them

https://github.com/seqan/seqan3/blob/bf271cca3e414b0c3c35d0fc248d91cbbeb3de54/test/unit/alphabet/quality_test.cpp#L49-L54

Reply:
Looking at the code

https://github.com/seqan/seqan3/blob/bf271cca3e414b0c3c35d0fc248d91cbbeb3de54/test/unit/alphabet/quality_test.cpp#L59-L74

I think it was not a `quality_test_template`, but a `phred_test_template` or a integration test between phreads

Reply:
remove below here: